src/dscanner/analysis/unused_label.d

```diff
@@ -147,13 +140,13 @@ private:
 			else if (!label.used)
 			{
 				addErrorMessage(label.line, label.column, "dscanner.suspicious.unused_label",
-						"Label \"" ~ label.name ~ "\" is not used.");
+						"Label \"" ~ to!string(label.name) ~ "\" is not used.");
 			}
 		}
 		stack.length--;
```

Comment:
Is the stack used anywhere? It seems that push and pop scope have the role of checking if a label was used or not.

Reply:
It is being used in `popScope` to iterate through labels and see if they are used or not

Reply:
But this is the only use that I see.